gammapy/time/tests/test_lightcurve.py

```diff
@@ -28,3 +35,61 @@ def test_lightcurve_fvar():
 def test_lightcurve_plot():
     lc = LightCurve.simulate_example()
     lc.plot()
+
+
+@requires_data('gammapy-extra')
```

Comment:
If I remember correctly, then `@requires_data` and `@requires_dependency` doesn't work on fixture functions.
They only work on test functions.

So remove the line `@requires_data('gammapy-extra')` from the fixture function here.